scss/components/_soe_dm_brand_bar.scss

```diff
@@ -41,3 +41,35 @@
     }
   }
 }
+
+a {
+  font-weight: 400;
+}
+
+.node-type-stanford-magazine-article {
+  #header {
+    &.header {
+      background: $white;
+      padding: 30px 0 0;
+      @include breakpoint-max(xx-small) {
+        min-height: 45px;
+        padding: 20px 0 0;
+      }
+
+      #site-title-first-line.site-title-uppercase a {
+        @include breakpoint-max(x-small) {
+          font-size: 28px;
+          margin-bottom: -5px;
+        }
+        @include breakpoint-max(xx-small) {
+          font-size: 18px;
+          margin-bottom: -3px;
+        }
+        @include breakpoint-max(xxx-small) {
+          font-size: 16px;
+          margin-bottom: -4px;
+        }
+      }
+    }
+  }
+}
```

Comment:
@minorwm 
This give us _two_ places to maintain the brand bar. What about just update the code starting at line 7?